content/service_dialogs/transform-vm.yml

```diff
@@ -18,6 +18,82 @@
       display_method_options:
       position: 0
       dialog_fields:
+      - name: tag_category
+        description: Tag category.
```

Comment:
Minor, but let's remove the `.` at the end of this description.  It does not match any of the others.